fastlane: convert en-GB to the default en-US and add DE locale

IzzySoft opened the pull request:
This PR adds short and full description for German (`de`) and renames `en-GB` to `en-US`, which is the fastlane default/fallback for missing locales.

IzzySoft committed:
fastlane: convert en-GB to the (mandatory) default en-US and add DE locale

oliexdev commented:
Thanks for the clarification and your project IzzyOnDroid. :+1: 
I have just updated a note in the full_description.txt (maybe fdroid updates it from time to time?) and I might also mark as archived but I need to read more what does this mean and if that can be undone (maybe in the future I will continoue working on that project or somebody else).

IzzySoft commented:
> I have just updated a note in the full_description.txt (maybe fdroid updates it from time to time?)

I have triggered a refresh here (please check after the next sync if all was carried over fine). As outlined, F-Droid only updates this when a new release is being pulled in.

> I might also mark as archived but I need to read more what does this mean and if that can be undone

It can, you can always unarchive. I used to do that back when I moved all my projects to Codeberg. On one of the projects, whenever I made a new release I unarchived it, so Codeberg could "forward" my push, then archived it again.

> maybe in the future I will continoue working on that project or somebody else

That would be cool! Give us a ping then should this happen, so we enable update checks again and remove the `SourceArchived` flag (not yet set as you did not yet archive – once you do, please give me a ping here as "last action before" :wink:)

oliexdev commented:
Thanks! I will archive this project until I continue working on it again.  
Regarding F-Droid, I’ve created a [merge request](https://gitlab.com/fdroid/fdroiddata/-/merge_requests/26270) to disable auto-updates.
